internal/core/src/exec/expression/UnaryExpr.cpp

```diff
@@ -1133,6 +1130,11 @@ PhyUnaryRangeFilterExpr::CanUseIndex() {
 
 bool
 PhyUnaryRangeFilterExpr::CanUseIndexForJson(DataType val_type) {
```

Comment:
what about term, binaryArith, and other expression like exist and json contains?

We need to quickly go through all the possible expressions support

Reply:
I'm ok if we want to finish in a secondary pr